splunk_event/test_splunk_event.py

```diff
@@ -1513,3 +1617,195 @@ def _mocked_auth_session_to_check_instance_config(instance):
             '_saved_searches': _mocked_saved_searches
         })
         self.assertEqual(len(self.events), 2)
+
+
+class TestSplunkEventsWithTokenAuth(AgentCheckTest):
```

Comment:
Comparing the methods with metrics, I miss:
test_check_audience_param_not_set
test_check_name_param_not_set
test_check_token_auth_preferred_over_basic_auth
test_check_token_renewed_when_almost_expired
test_check_memory_token_expired

Do we need these here as well?

Reply:
@whazenberg  Yeah, I just put in `metrics` to verify because both have a common implementation, so it will work. If you say then I can put those tests here as well.

Reply:
Depends on what we did before, for other cases with metrics/events. If it's uncommon to duplicate then let's not. I was more thinking along the lines of when one or the other is split up in any way we might miss some test cases.